### PR TITLE
init UI of the Main screen

### PR DESCRIPTION
**Pull request by thaohq-0814**
Init UI of the main screen
Link Redmine: https://edu-redmine.framgia.vn/issues/5381

### CODE REVIEW

File: app/src/main/res/drawable/ic_tab_audio.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    <path android:fillColor="#FF000000" android:pathData="m80,240c8.832,0 16,-7.168 16,-16v-64c0,-8.832 -7.168,-16 -16,-16s-16,7.168 -16,16v64c0,8.832 7.168,16 16,16zM80,240"/>
+    <path android:fillColor="#FF000000" android:pathData="m264,256v-128c0,-8.832 -7.168,-16 -16,-16s-16,7.168 -16,16v128c0,8.832 7.168,16 16,16s16,-7.168 16,-16zM264,256"/>
+    <path android:fillColor="#FF000000" android:pathData="m368,96c-8.832,0 -16,7.168 -16,16v160c0,8.832 7.168,16 16,16s16,-7.168 16,-16v-160c0,-8.832 -7.168,-16 -16,-16zM368,96"/>
+    <path android:fillColor="#FF000000" android:pathData="m304,144c-8.832,0 -16,7.168 -16,16v64c0,8.832 7.168,16 16,16s16,-7.168 16,-16v-64c0,-8.832 -7.168,-16 -16,-16zM304,144"/>
+    <path android:fillColor="#FF000000" android:pathData="m176,368c0,8.832 7.168,16 16,16s16,-7.168 16,-16v-16c0,-8.832 -7.168,-16 -16,-16s-16,7.168 -16,16zM176,368"/>
+    <path android:fillColor="#FF000000" android:pathData="m192,48c8.832,0 16,-7.168 16,-16v-16c0,-8.832 -7.168,-16 -16,-16s-16,7.168 -16,16v16c0,8.832 7.168,16 16,16zM192,48"/>
+</vector>
```
Comment (buivanhieu7112) [inline app/src/main/res/drawable/ic_tab_audio.xml:33]: reformat code

File: app/src/main/res/drawable/ic_tab_home.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="306.773"
+    android:viewportWidth="306.773" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#010002" android:pathData="M302.93,149.794c5.561,-6.116 5.024,-15.49 -1.199,-20.932L164.63,8.898c-6.223,-5.442 -16.2,-5.328 -22.292,0.257L4.771,135.258c-6.092,5.585 -6.391,14.947 -0.662,20.902l3.449,3.592c5.722,5.955 14.971,6.665 20.645,1.581l10.281,-9.207v134.792c0,8.27 6.701,14.965 14.965,14.965h53.624c8.264,0 14.965,-6.695 14.965,-14.965v-94.3h68.398v94.3c-0.119,8.264 5.794,14.959 14.058,14.959h56.828c8.264,0 14.965,-6.695 14.965,-14.965V154.024c0,0 2.84,2.488 6.343,5.567c3.497,3.073 10.842,0.609 16.403,-5.513L302.93,149.794z"/>
+</vector>
```
Comment (buivanhieu7112) [inline app/src/main/res/drawable/ic_tab_home.xml:9]: as above

File: app/src/main/res/drawable/ic_tab_library.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<vector android:height="24dp" android:viewportHeight="612.009"
+    android:viewportWidth="612.009" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M507.689,607.106l83.357,-22.326c15.093,-4.059 24.098,-19.467 19.952,-35.018L470.441,25.265C466.354,9.972 450.574,0.91 435.681,4.912l-83.356,22.326c-15.122,4.059 -24.099,19.467 -19.953,35.018L472.9,586.753C477.017,602.047 492.768,611.08 507.689,607.106z"/>
+    <path android:fillColor="#FF000000" android:pathData="M114.602,606.163c15.637,0 28.329,-12.578 28.329,-28.672V34.499c0,-15.837 -12.893,-28.644 -28.329,-28.644H28.329C12.692,5.855 0,18.404 0,34.499v542.993c0,15.837 12.893,28.644 28.329,28.644h86.273V606.163z"/>
+    <path android:fillColor="#FF000000" android:pathData="M286.118,606.163c15.637,0 28.329,-12.55 28.329,-28.644V34.499c0,-15.837 -12.893,-28.644 -28.329,-28.644h-86.272c-15.637,0 -28.329,12.549 -28.329,28.644v542.993c0,15.837 12.893,28.644 28.329,28.644h86.272V606.163z"/>
+</vector>
```
Comment (buivanhieu7112) [inline app/src/main/res/drawable/ic_tab_library.xml:15]: as above

File: app/src/main/res/drawable/ic_tab_search.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="56.966"
+    android:viewportWidth="56.966" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M55.146,51.887L41.588,37.786c3.486,-4.144 5.396,-9.358 5.396,-14.786c0,-12.682 -10.318,-23 -23,-23s-23,10.318 -23,23s10.318,23 23,23c4.761,0 9.298,-1.436 13.177,-4.162l13.661,14.208c0.571,0.593 1.339,0.92 2.162,0.92c0.779,0 1.518,-0.297 2.079,-0.837C56.255,54.982 56.293,53.08 55.146,51.887zM23.984,6c9.374,0 17,7.626 17,17s-7.626,17 -17,17s-17,-7.626 -17,-17S14.61,6 23.984,6z"/>
+</vector>
```
Comment (buivanhieu7112) [inline app/src/main/res/drawable/ic_tab_search.xml:9]: as above

**Review by buivanhieu7112 — APPROVED** [review]

**Comment by thaohq-0814** [comment]
ready

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -1,8 +1,33 @@
...
+        app:itemTextColor="@color/bottom_navigation_item_state"
+        app:labelVisibilityMode="labeled"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:menu="@menu/bottom_navigation_activity_main"
+        />
+
+    <FrameLayout
+        android:id="@+id/layout_container"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]:
```suggestion
        android:id="@+id/layoutContainer"
```

File: app/src/main/res/menu/bottom_navigation_activity_main.xml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/action_home"
```
Comment (tuanlvt-1117) [inline app/src/main/res/menu/bottom_navigation_activity_main.xml]:
```suggestion
        android:id="@+id/actionHome"
```

File: app/src/main/res/menu/bottom_navigation_activity_main.xml [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+        android:icon="@drawable/ic_tab_home"
+        android:title="@string/tab_home"
+        app:showAsAction="ifRoom"/>
+    <item
+        android:id="@+id/action_audio"
```
Comment (tuanlvt-1117) [inline app/src/main/res/menu/bottom_navigation_activity_main.xml]:
```suggestion
        android:id="@+id/actionAudio"
```

File: app/src/main/res/menu/bottom_navigation_activity_main.xml [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        android:icon="@drawable/ic_tab_audio"
+        android:title="@string/tab_audio"
+        app:showAsAction="ifRoom"/>
+    <item
+        android:id="@+id/action_search"
```
Comment (tuanlvt-1117) [inline app/src/main/res/menu/bottom_navigation_activity_main.xml]:
```suggestion
        android:id="@+id/actionSearch"
```

File: app/src/main/res/menu/bottom_navigation_activity_main.xml [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+        android:icon="@drawable/ic_tab_search"
+        android:title="@string/tab_search"
+        app:showAsAction="ifRoom"/>
+    <item
+        android:id="@+id/action_library"
```
Comment (tuanlvt-1117) [inline app/src/main/res/menu/bottom_navigation_activity_main.xml]:
```suggestion
        android:id="@+id/actionLibrary"
```

File: app/src/main/res/values/dimens.xml [hunk 10]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dimen_0">0dp</dimen>
```
Comment (tuanlvt-1117) [inline app/src/main/res/values/dimens.xml]: 0dp không cần đưa vào dimen cũng được e

**Comment by thaohq-0814** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**